tasks/get_objects_for_app.yml

```diff
@@ -6,6 +6,16 @@
     templates: "{{ app | json_query('services[*].templates[]') | list }}"
   tags: deploy
 
+- name: Set OpenShift datadog objects to manage
+  set_fact:
+    images: "{{ templates | map('regex_search', '.*/is.yml.j2$') | select('string') | list }}"
+    builds: "{{ templates | map('regex_search', '.*/bc.yml.j2$') | select('string') | list }}"
+  when: ddtrace
```

Comment:
We always want to collect those objects not only when `ddtrace` is true. They are already conditionally created _via_ their templates.